chart/openfaas/values.yaml

```diff
@@ -309,7 +309,7 @@ ingress:
 # ingressOperator (optional) – component to have specific FQDN and TLS for Functions
 # https://github.com/openfaas-incubator/ingress-operator
 ingressOperator:
-  image: ghcr.io/openfaas/ingress-operator:0.7.1
+  image: ghcr.io/openfaas/ingress-operator:0.8.0
```

Comment:
The CRD also changed and will need to be imported. You should see this fail if you test out the chart with the ingress operator.

Reply:
Updated the ingress-operator CRD.